App/Sources/Presentation/StudentMain/Source/Scene/StudentMainView.swift

```diff
@@ -1,9 +1,67 @@
 import SwiftUI
 
+import PopupView
+
 struct StudentMainView: View {
+    @StateObject var studentMainContainer: MVIContainer<StudentMainIntent, StudentMainModelStateProtocol>
+    
+    @State private var showPopup = false
+    
     var body: some View {
-        VStack(spacing: 0) {
-            Text("Main View")
+        NavigationView {
+            VStack {
+                BannerView()
+                
+                Text("* 문제는 12:30 ~ 19:30분까지 풀수 있습니다.")
+                    .skFont(.pr12)
+                    .frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
```suggestion
                    .frame(maxWidth: .infinity, alignment: .leading)
```
Vstack에 alignment: .leading을 주고 전체에 padding(.horizontal, 16)을 주면 중복되는 부분도 줄고 이렇게 .fram으로 정렬시켜주지 않아도 될 것 같은데 시도 해보셨나요!?

Reply:
아 죄송합니다.. 리뷰에다가 한다고 해놓고 안한 부분이 있었네요... 죄송합니다.. 다시 시도해보도록 하겠습니다 ㅜㅜㅜ

Reply:
근데 제가 생각해보니까 저렇게 한 이유가 bannerView 같은 경우에는 horizontal 없이 적용되서 따로 저렇게 둔거였습니다. 지금 horizontal이 적용되는 부분은 pr에 올라온 영상 부분에서 banner아래 두개의 scrollView에서 적용시키고 있습니다!!
두개를 따로 묶어서 horizontal을 적용시킬 수 있을 거 같습니다만 그렇게 적용시킬까요 아니면 그대로 둘까요?? 의견을 여쭤보고 싶습니다~

Reply:
음 저는 반응형 생각하면 NavigationView 안에 ScrollView를 하나 더 만들어서 진행하면 좋을 거 같아요!!

`
NavigationView {
    ScrollView {
        BannerView()
        
        Vstack(alignment: .leading) {
            ... ViewCode
        }
        .padding(.horizontal, 16)    
`

급하게 작성해서 맞는 지는 모르겠는데 이런 느낌으로 가는 거 어떨까요?

Reply:
오호~ 그런 방법도 있었군요~ 한번 참고해서 다시 작성하여 올려보도록 하겠습니다~